invenio_app_ils/ui/src/invenio_app_ils/common/api/eitems/__tests__/eitem.js

```diff
@@ -0,0 +1,22 @@
+import { eitem as eitemApi } from '../eitem';
+
+describe('EItem query builder tests', () => {
```

Comment:
These tests for the query builder can be written once, testing the query builder itself. Like this we won't have to replicate them for every model (document, item, loan .. etc.)